app/src/pages/playground/PlaygroundInstance.tsx

```diff
@@ -28,6 +29,7 @@ export function PlaygroundInstance(props: PlaygroundInstanceProps) {
   return (
     <PanelGroup direction={isSingleInstance ? "horizontal" : "vertical"}>
       <Panel defaultSize={50} order={1} css={panelContentCSS}>
+        <PlaygroundInstanceErrors {...props} />
```

Comment:
I think we eventually put this in a banner as a single "warning". Can always refactor